Simplify installation instructions

landermkerbey opened the pull request:
As we discussed previously, this proposed adjustment prevents an unwieldy redirect of the reader's attention to a prior command by unifying the discussion of both separate installation and integrated installation into the paragraph between the two shell code blocks. This also makes the document more concise.

landermkerbey committed:
Simplify installation instructions

As we discussed previously, this proposed adjustment prevents an unwieldy redirect of the reader's attention to a prior command by unifying the discussion of both separate installation and integrated installation into the paragraph between the two shell code blocks. This also makes the document more concise.

jambonrose commented:
Thanks for the work!

Would you like to add yourself to the `AUTHORS.rst` file?

landermkerbey commented:
I'll go ahead and amend AUTHORS.rst. I should be able to go over the rest of the docs today and tomorrow.

jambonrose commented on docs/quickstart.rst:
```diff
@@ -22,16 +22,13 @@ In a Terminal, use :code:`pip` to install the package from `PyPI`_.
 If you intend to use the :class:`~improved_user.factories.UserFactory`
 provided by the package to allow for testing with |factory_boy|_, you
-can specify so during install.
+can either install |factory_boy| separately, or use the modified installation
```
This RST directive needs a `_` at the end of it (like the one on the previous line).

Once you [build the docs](http://django-improved-user.readthedocs.io/en/latest/contributing.html#your-first-documentation-contribution), hover over items that should be linked to verify they work as desired. Sphinx does not provide errors for items like this, unfortunately.

landermkerbey replied:
I agree. I think it's reasonable to assume that the standard user for this project is more likely to want the factory inclusion than not. I'll amend this to reflect that.

jambonrose reviewed (changes requested):
I like that this tightens things up. I think we can be more aggressive, though.

Why not start with the install command with the factory, and then note that if the factory is unnecessary, it can be omitted?

landermkerbey committed:
Update quickstart.rst

While the overall word count is now fairly similar to that of the original document, the flow of the initial instruction is more logical and targeted towards the likely preferences of this project's expected end-user.

jambonrose committed:
Merge branch 'development' into patch-1
